tinygrad/runtime/ops_metal.py

```diff
@@ -176,7 +196,7 @@ def __init__(self, device:str):
     self.timeline_value = 0
 
     from tinygrad.runtime.graph.metal import MetalGraph
-    super().__init__(device, MetalAllocator(self), MetalRenderer(), MetalCompiler() if getenv("METAL_XCODE") else Compiler(),
+    super().__init__(device, MetalAllocator(self), MetalRenderer(), MetalCompiler() if getenv("METAL_DIRECT", 1) else Compiler(), \
```

Comment:
It's little super frustrating things. Why did you add a \ here? Like read the diff over and over.